test/llvm/CMakeLists.txt

```diff
@@ -35,9 +35,14 @@ find_library(LLVMSupport_LIBRARY NAMES LLVMSupport PATHS ${LLVM_LIBRARY_DIR})
 
 message(STATUS "adding hiptensor component: hiptensor_llvm")
 message(STATUS "LLVM_LIBRARY_DIR: ${LLVM_LIBRARY_DIR}")
+message(STATUS "LLVM_VERSION_MAJOR: ${LLVM_VERSION_MAJOR}")
 message(STATUS "LLVMObjectYAML_LIBRARY: ${LLVMObjectYAML_LIBRARY}")
 message(STATUS "LLVMSupport_LIBRARY: ${LLVMSupport_LIBRARY}")
 
```

Comment:
Found that llvm::raw_fd_ostream(string & const filename, error_code ec) is only supported by llvm 7.0 and above